opdracht1.js

```diff
@@ -5,32 +5,48 @@ const tvTypes = inventory.map((item) => {
     return item.type;
 });
 
-console.log(tvTypes);
+//console.log(tvTypes); <-- console.log van opdracht 1
 
 
 //Opdracht 1b
 //Maak een nieuwe array en filter op originalStock - Sold > 0
 //Indien true voeg toe aan nieuwe array
-const soldOutTvs = inventory.filter((item) => {
-    return (item.originalStock - item.sold) > 0;
-});
+//voor de bonusopdracht zet het geheel in een functie
+
+function getSoldOutTvs(){
+    const soldOutTvs = inventory.filter((item) => {
+        return (item.originalStock - item.sold) <= 0;
+    });
```

Comment:
Je zou dit nog iets kunnen simplificeren, want wanneer het aantal verkochte tv's gelijk is aan de originele voorraad, dan betekent dat óók dat ze uitverkocht zijn
```suggestion
    const soldOutTvs = inventory.filter((item) => {
        return item.originalStock === item.sold;
    });
```
    
En je zou het nog iets verder kunnen versimpelen:
```suggestion
    const soldOutTvs = inventory.filter(item => item.originalStock === item.sold);